sycl/include/sycl/ext/oneapi/matrix/matrix-jit.hpp

```diff
@@ -477,8 +477,8 @@ class wi_element<uint16_t, NumRows, NumCols, Layout, Group> {
 
   explicit operator bool() {
 #ifdef __SYCL_DEVICE_ONLY__
-    return __spirv_VectorExtractDynamic(M.spvm, idx) !=
-           static_cast<uint16_t>(0);
+    return std::fabs(make_fp32(__spirv_VectorExtractDynamic(M.spvm, idx))) <
```

Comment:
it seems element_wise_all_ops_bf16_yubing_20220128.cpp failed on this line.
when I change to
make_fp32(__spirv_VectorExtractDynamic(M.spvm, idx)) != 0.0f;
the testcase can pass

Reply:
we use ">=" instead of "<"